lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -1,20 +1,32 @@
 <div class="section">
     <div class="course-meta-container col-12 md-col-8 sm-col-12">
-        <a href="<%- course_url %>" class="course-image-link">
-            <img
-                class="header-img"
-                src="<%- course_image_url %>"
-                alt="<%= interpolate(gettext('%(courseName)s Home Page.'), {courseName: display_name}, true) %>"/>
-        </a>
+        <div class="course-image-container">
+            <% if (course_url){ %>
+                <a href="<%- course_url %>" class="course-image-link">
+            <% } %>
+                <img
+                    class="header-img"
+                    src="<%- course_image_url %>"
+                    alt="<%= interpolate(gettext('%(courseName)s Home Page.'), {courseName: display_name}, true) %>"/>
```

Comment:
Why is that? I thought regardless the image is a link or not, the alt-text will show when user hovers the mouse over the image. Right?